src/pages/Tags/Tags.tsx

```diff
@@ -1,9 +1,242 @@
+        map(tagCategorySelection, (tagCategoryValue) => {
+          return {
+            type: tagCategoryValue,
+            name: tagInputValue,
+          }
+        })
+      )
+      const payload: TagsType = {
+        tags: transformedObject,
+      }
+
+      try {
+        await createTags(payload)
+        showNotification({
+          type: NotificationTypes.Success,
+          title: t('notification.announcement'),
+          content: t('success.tag_added'),
+        })
+        reset()
+      } catch (errorData) {
+        showValidationErrorMessage(errorData)
+      }
+    },
+    [createTags, reset, t]
+  )
+
+  const groupedCategoryData = groupBy(tags, 'type')
+  const uniqueCategoryTypes = Object.keys(groupedCategoryData)
+
+  const areInputFieldsDirty = !!(
+    dirtyFields.tagCategorySelection && dirtyFields.tagInput
+  )
+
+  if (isFetchingTags) {
+    return <Loader loading />
+  }
+
+  return (
+    <>
+      <div className={classes.tagsHeader}>
+        <h1>{t('tag.tag_management')}</h1>
+        <Tooltip
+          title={t('cheat_sheet.user_management.title')}
+          modalContent={<TagsCheatSheet />}
+        />
+      </div>
+      <Container className={classes.tagsContainer}>
+        <div>
+          <h4 className={classes.addingTag}>{t('tag.adding_tag')}</h4>
+          <p>{t('tag.naming_tag')}</p>
+        </div>
+
+        <div className={classes.addingTagsSeparator} />
+
+        <div className={classes.tagsSection}>
+          <DynamicForm fields={inputFields} control={control} />
+          <Button
+            appearance={AppearanceTypes.Text}
+            iconPositioning={IconPositioningTypes.Left}
+            icon={Add}
+            className={classes.addNewRow}
+            children={t('tag.add_new_row')}
+            onClick={addInputField}
+            hidden={!includes(userPrivileges, Privileges.AddTag)}
+          />
+        </div>
+
+        <div className={classes.categorySection}>
+          <DynamicForm fields={categoryFields} control={control} />
+          <Button
+            children={t('button.add')}
+            type="submit"
+            ariaLabel={t('button.add')}
+            className={classes.addButton}
+            onClick={handleSubmit(onTagsSubmit)}
+            loading={isCreatingTags}
+            disabled={!areInputFieldsDirty}
+            hidden={!includes(userPrivileges, Privileges.AddTag)}
+          />
+        </div>
+      </Container>
+
+      <div className={classes.categoryContainer}>
+        {map(uniqueCategoryTypes, (type) => (
+          <Container key={type} className={classes.category}>
+            <span className={classes.tagCategoryNameContainer}>
+              <div className={classes.categoryName}>{type}</div>
+              <Button
+                appearance={AppearanceTypes.Text}
+                icon={EditIcon}
+                className={classes.editIcon}
+                hidden={type === t('tag.skills_tag')}
```

Comment:
**[blocker]**

This is extremely error-prone and will cause an issue right away in any other language. You are checking a constant value (type) against a translatable value, meant only for displaying (t('tag.skills_tag')). This will break as soon as the user switched to another language, or the client decides to use some other word for displaying "tag.skills_tag" text somewhere